### PR TITLE
Add webhook example

### PR DESCRIPTION
**Pull request by NotNorom**
I'd like to add this example because I needed some time to figure out you actually don't need a token if you are only doing webhooks and so others won't have to.

### CODE REVIEW

File: examples/e18_webhook/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+use serenity::http::HttpBuilder;
+
+#[tokio::main]
+async fn main() {
+    // You don't need a token when you are only dealing with webhooks
```
Comment (DRuppFv) [inline examples/e18_webhook/src/main.rs]: add a "." here, in the ending of the comment.
add a "." here, in the ending of the comment.
add a "." here, in the ending of the comment.

File: examples/e18_webhook/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    let http = HttpBuilder::new("").build();
```
Comment (mkrasnitski) [inline examples/e18_webhook/src/main.rs]:
```suggestion
    let http = Http::new("");
```
You don't need to use `HttpBuilder` if you're only passing in a token. Will need to change the import as well.

File: examples/e18_webhook/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    let webhook = http.get_webhook_from_url("https://discord.com/api/webhooks/133742013374206969/hello-there-oPNtRN5UY5DVmBe7m1N0HE-replace-me-Dw9LRkgq3zI7LoW3Rb-k-q").await.expect("Replace the webhook with your own");
```
Comment (mkrasnitski) [inline examples/e18_webhook/src/main.rs]:
```suggestion
    let webhook = Webhook::from_url(&http, "https://discord.com/api/webhooks/133742013374206969/hello-there-oPNtRN5UY5DVmBe7m1N0HE-replace-me-Dw9LRkgq3zI7LoW3Rb-k-q").await.expect("Replace the webhook with your own");
```

More ergonomic to construct the Webhook directly. Will need to import it as well.

File: examples/e18_webhook/src/main.rs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+use serenity::{http::Http, model::webhook::Webhook};
```
Comment (DRuppFv) [inline examples/e18_webhook/src/main.rs]: Making cargo format happy!!
```suggestion
use serenity::http::Http;
use serenity::model::webhook::Webhook;
```

**Review by arqunis — APPROVED** [review]

**Comment by vicky5124** [comment]
The PR was merged without the example being added in the root `/Makefile.toml`, nor in `/.github/workflows/ci.yml`

**Comment by nickelc** [comment]
> The PR was merged without the example being added in the root `/Makefile.toml`, nor in `/.github/workflows/ci.yml`

Fixed in [7cec025](https://github.com/serenity-rs/serenity/commit/7cec025afabd4e03cc40c9f350a779e81429aeee)

**Pull request merged**